shell_completion/README.md

```diff
@@ -0,0 +1,9 @@
+# gaiad
```

Comment:
Could you please move `shell_completion` folder to `./contrib` folder of the gaia repo?